cli/args/lockfile.rs

```diff
@@ -236,6 +237,54 @@ impl CliLockfile {
           ))
         })
         .collect(),
+      patches: if workspace.has_unstable("npm-patch") {
```

Comment:
This requires putting this in the deno.json. I think that's fine because the patch feature needs to be in a deno.json and I think it's not worth it to support `--unstable-npm-patch` because this is only going to exist for a few weeks.